Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -925,33 +925,48 @@ def related_incidents(incident_id):
 
 
 def fetch_incidents():
-    now = int(time.time())
     last_run = demisto.getLastRun() and demisto.getLastRun().get('time')
     if not last_run:
-        now -= 600  # In the first run, fetch incidents from the last 10 minutes
-        last_run = now
-    args = {'date-created-after': normalize_timestamp(last_run * 1000)}
+        last_run = date_to_timestamp(FETCH_TIME)
```

Comment:
`FETCH_TIME` will be initiated to `''` if the param is not decided by the user.
I think that would make this call fail.

Reply:
Will be verified in the test module so shouldn't happen